test-harness/src/main/java/edu/umd/cs/findbugs/test/CountMatcher.java

```diff
@@ -33,7 +35,25 @@ public CountMatcher(int count, Matcher<T> matcher) {
      * @return new matcher instance
      */
     public static <T> Matcher<Iterable<T>> containsExactly(final int count, final Matcher<T> matcher) {
-        return new CountMatcher<>(count, matcher);
+        return new CountMatcher<>(count, count, matcher);
+    }
+
+    /**
+     * Creates a matcher for {@link Iterable}s that only matches if at least {@code minCount} and at most {@code maxCount}
+     * items match the specified {@code matcher}.
+     *
+     * @param matcher
+     *            A non-{@code null} matcher that must match at least {@code minCount} and at most {@code maxCount} times.
+     * @param <T>
+     *            matcher item type
+     * @param minCount
```

Comment:
In some contexts the upper (or lower) bound of a "between" might be exclusive, could you please add to the comment that they are both inclusive?

Reply:
Thanks, added this info to the javadoc.

Reply:
Thank you!